core/opendaq/modulemanager/src/context_impl.cpp

```diff
@@ -304,6 +304,10 @@ void ContextImpl::registerOpenDaqTypes()
                                     .build();
 
     typeManager->addType(ptpSyncInterface);
+
```

Comment:
Could you please also move the registration of `"ConnectionStatusType"` to this method instead of registering it directly in the constructor? I initially overlooked the fact that there’s a `registerOpenDaqTypes` method, which is a more appropriate place for such registration.

Reply:
I moved it and also removed additional `hasType` check, since this is only called once per Context.